### PR TITLE
Add TCB 67 about Exasol

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _episodes/67.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+* Enable HTTP/2 for internal communication by default.
+* Add `timezone()` functions.
+* Include table functions with `SHOW FUNCTIONS` output.
+* Add support for writing change data feed when deletion vector is enabled to
+  the Delta Lake connector.
+
+[Trino 464](https://trino.io/docs/current/release/release-464.html)
+
+* Require JDK 23 to run Trino.
+* Add the Faker connector.
```
Comment (colebow) [inline _episodes/67.md:61]: I will be making a reference to the League of Legends player who goes by Faker. Fair warning.

File: _episodes/67.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+* Add the Vertica connector.
+* Remove the Accumulo connector.
+
+As usual, numerous performance improvements, bug fixes, and other features were
```
Comment (colebow) [inline _episodes/67.md:65]:
```suggestion
As always, numerous performance improvements, bug fixes, and other features were
```
We can be even more explicit about this.

File: _episodes/67.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+added as well.
+
+* Trino maintainer call - great sync with some exciting news coming to the community soon. 
+* Trino contributor call - [recording and minutes available now](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-call-24-oct-2024). 
+* Trino K8s operator meeting - minutes coming soon.
```
Comment (colebow) [inline _episodes/67.md:70]: Spell out Kubernetes?

File: _episodes/67.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+* Trino Summit call for speakers closed - stay tuned for announcements.
```
Comment (colebow) [inline _episodes/67.md:71]: Maybe let's highlight that Trino Summit is coming soon and people should register, so there's a clearer call to action.

File: _episodes/67.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+## Introducing Thomas and Exasol
+
+Exasol is a lightning fast, in-memory database for analytics.
```
Comment (colebow) [inline _episodes/67.md:75]: You could say it is _literally_ the fastest, given that it's been at the top of TPC-H benchmarks for like two decades. Fun thing to call out, I think.

File: _episodes/67.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+## Introducing Thomas and Exasol
+
+Exasol is a lightning fast, in-memory database for analytics. 
+
+<img src="{{site.baseurl}}/assets/images/logos/exasol-small.png">
+
+## Exasol and Trino
+
+Trino can Exasol bridge the gap between extreme performance with in-memory usage
```
Comment (colebow) [inline _episodes/67.md:81]:
```suggestion
Trino and Exasol bridge the gap between extreme performance with in-memory usage
```

File: _episodes/67.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+- time: 0:00
+  title: Introduction
+- time: 0:53
+  title: Trino releases 461, 462, and 463
+- time: 9:30
+  title: Guests introductions
+- time: 1:00:42
+  title: Upcoming events and rounding out
+introduction: |
+  Manfred is joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,
```
Comment (colebow) [inline _episodes/67.md:18]:
```suggestion
  Manfred and Cole are joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,
```
:)

File: _episodes/67.md (hunk 1 above)
Comment (mosabua) [reply _episodes/67.md:61]: All good .. I wont have a clue what you are talking about ;-)

Comment (mosabua) [reply _episodes/67.md:61]: fyi @nineinchnick

File: _episodes/67.md (hunk 3 above)
Comment (mosabua) [reply _episodes/67.md:70]: 👍

File: _episodes/67.md (hunk 5 above)
Comment (mosabua) [reply _episodes/67.md:75]: Sounds good.

File: _episodes/67.md (hunk 7 above)
Comment (mosabua) [reply _episodes/67.md:18]: Oopps.. thats from before I knew if you would join ;-)

File: _episodes/67.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+- time: 0:00
+  title: Introduction
+- time: 0:53
+  title: Trino releases 461, 462, and 463
+- time: 9:30
+  title: Guests introductions
+- time: 1:00:42
+  title: Upcoming events and rounding out
+- time: 0:00
+  title: Introduction
```
Comment (colebow) [inline _episodes/67.md:18]:
```suggestion
```

Comment (colebow) [reply _episodes/67.md:18]: Or whatever needs fixing here...

File: broadcast/index.md [hunk 9]
```diff
@@ -20,12 +20,8 @@ interesting developments in the ecosystem around Trino.
...
 
 <dl>
-<dt>31 October 2024: Trino Community Broadcast 67 - Extra speed with Exasol and Trino</dt>
-<dd>
-<a href="https://www.linkedin.com/in/thomas-bestfleisch/">Thomas Bestfleisch</a>
-from <a href="{{site.url}}/ecosystem/data-source.html#exasol">Exasol</a> join us to
-talk about advantages and scenarios of using the in-memory, column-oriented RDBMS Exasol with Trino.</dd>
-
+<dt>14 November 2024: Trino Community Broadcast 68</dt>
+<dd>Topic to be determined - contact us if you are interested to join as guest</dd>
```
Comment (colebow) [inline broadcast/index.md:24]:
```suggestion
<dd>Topic to be determined - contact us if you are interested to join as a guest.</dd>
```

File: _episodes/67.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+the opportunity to question him for more details.
+
+The pull request for the Exasol connector has been a long time in the works and
+was finally merged for Trino 452. We talk about the motivation, the process,
+the results, and the future for the connector.
+
+## Resources
+
+* [Exasol](https://www.exasol.com/)
+* [Trino connector](https://trino.io/docs/current/connector/exasol.html)
```
Comment (colebow) [inline _episodes/67.md:114]:
```suggestion
* [Trino's Exasol connector](https://trino.io/docs/current/connector/exasol.html)
```

**Review by colebow — APPROVED** [review]
LGTM pending comments

File: broadcast/index.md [hunk 11]
```diff
@@ -20,12 +20,8 @@ interesting developments in the ecosystem around Trino.
 ## Upcoming episodes
 
 <dl>
-<dt>31 October 2024: Trino Community Broadcast 67 - Extra speed with Exasol and Trino</dt>
-<dd>
-<a href="https://www.linkedin.com/in/thomas-bestfleisch/">Thomas Bestfleisch</a>
-from <a href="{{site.url}}/ecosystem/data-source.html#exasol">Exasol</a> join us to
-talk about advantages and scenarios of using the in-memory, column-oriented RDBMS Exasol with Trino.</dd>
-
+<dt>21 November 2024: Trino Community Broadcast 68</dt>
```
Comment (mosabua) [inline broadcast/index.md:23]: fyi @colebow @bitsondatadev - I am not available for 14th .. so going for 21st .. topic yet to be determined ..

**Pull request merged**